tests/gramex.yaml

```diff
@@ -1171,7 +1170,6 @@ url:
       data:
         url: $YAMLPATH/../testlib/iris.csv
       model:
-        path: $YAMLPATH/model.pkl
```

Comment:
ToDo: Change guide to reflect that paths to saved models are no longers directly settable. Only the path to a config directory can be set, where the model (and other config) will be saved.